src/app/inventory/item-types.ts

```diff
@@ -412,3 +412,23 @@ export interface DimPursuit {
   /** Modifiers active in this quest */
   modifierHashes: number[];
 }
+
+export interface DimAdjustedItemPlug {
```

Comment:
One minor thing, can we pull this out of item-types.ts and stick it into a types.ts inside the compare folder since that is the only place they are used?